sound/soc/sof/intel/hda-dai.c

```diff
@@ -183,16 +183,15 @@ static struct sof_ipc_dai_config *hda_dai_update_config(struct snd_soc_dapm_widg
 	return config;
 }
 
-static int hda_dai_widget_update(struct sof_intel_hda_stream *hda_stream,
+static int hda_dai_widget_update(struct device *dev,
 				 struct snd_soc_dapm_widget *w,
 				 int channel, bool widget_setup)
 {
-	struct snd_sof_dev *sdev = hda_stream->sdev;
 	struct sof_ipc_dai_config *config;
 
 	config = hda_dai_update_config(w, channel);
 	if (!config) {
-		dev_err(sdev->dev, "error: no config for DAI %s\n", w->name);
+		dev_err(dev, "error: no config for DAI %s\n", w->name);
```

Comment:
Can we drop the "error: " prefix at the same time?

Reply:
done

Reply:
done